charts/govuk-apps-conf/templates/external-secrets/publishing-api/ec2-database-url.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: external-secrets.io/v1alpha1
+kind: ExternalSecret
+metadata:
+  name: publishing-api-database-url-ec2
+  labels:
+    {{- include "govuk-apps-conf.labels" . | nindent 4 }}
+  annotations:
+    a8r.io/description: >
+      Client key for Sentry publishing-api project.
+spec:
+  refreshInterval: {{ .Values.externalSecrets.refreshInterval }}
+  secretStoreRef:
+    name: aws-secretsmanager
+    kind: ClusterSecretStore
+  target:
+    name: publishing-api-database-url-ec2
+  dataFrom:
+    - key: govuk/publishing-api/database-url
```

Comment:
was this stored properly? I think I already did one here: https://github.com/alphagov/govuk-helm-charts/blob/6dc2c34d819c71806e0931a4abb1a5bfe56e7540/charts/publishing-api/templates/common/database-external-secret.yaml

Reply:
Yeah I added it manually.